utilities.h

```diff
@@ -171,10 +163,10 @@ class Utilities
         vector<uint8_t> combined;
 
         // Calculate the total size of the combined array using a fold expression
-        size_t totalSize = (arrays.size() + ... + 0);
+        size_t totalSize = (arrays.size() + ...);
         combined.reserve(totalSize);
 
-        // Append each array to the combined vector using a fold expression
+        // Append each array to the combined vector using a comma fold expression
```

Comment:
I'm going to put the + 0 back as it was intentional - if you pass an empty array, it'll still work with the + 0, but otherwise has no anchor.  Let's discuss as needed!